xskillscore/tests/conftest.py

```diff
@@ -179,6 +179,19 @@ def weights_cos_lat(a):
     return xr.ones_like(a) * np.abs(np.cos(a.lat))
 
 
+@pytest.fixture
+def weights_linear_time(a):
+    """Weighting array by linear (1 -> 0) of the time."""
+    weights = np.linspace(1, 0, num=len(a.time))
+    return xr.ones_like(a) * xr.DataArray(weights, dims="time")
```

Comment:
I am unsure whether such `weights` are useful. shouldnt we only do `weight`ing across what we specific in `dim`?

Reply:
I would only use this `weights` if I'd want to get a metric over `time`,`lon` and `lat`. but as `lon` and `lat` are identical for all values, it's the same as just `weights_linear_time_1d`